### PR TITLE
Serenity updates - October 2020 - Titles and structure

### PR DESCRIPTION
**Pull request by NolwennP**

### CODE REVIEW

File: content/updates/2020-10/1-Interrupt-your-imports-and-exports.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Interrupt your imports and exports
```
Comment (LaureBro) [inline content/updates/2020-10/1-Interrupt-your-imports-and-exports.md:9]:
```suggestion
# More control over imports and exports
```

File: content/updates/2020-10/2-Rules-automatic-description-creation-and-new-efficient-filters.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+
+# Rules: automatic description creation and new efficient filters
```
Comment (LaureBro) [inline content/updates/2020-10/2-Rules-automatic-description-creation-and-new-efficient-filters.md:8]:
```suggestion
# A new rules engine action and new filters for conditions
```

File: content/updates/2020-10/2-Rules-automatic-description-creation-and-new-efficient-filters.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## A new action to automatize your description creation
+::: meta-data type="New" features="Automation, Productivity" available="early October" link-to-doc="../articles/.html"
+:::
+
+## New filters in conditions
```
Comment (LaureBro) [inline content/updates/2020-10/2-Rules-automatic-description-creation-and-new-efficient-filters.md:14]:
```suggestion
## New filters for conditions
```
? not sure if it's "in" or "for"

File: content/updates/2020-10/3-A-better-product-grid-for-product-models-and-variants.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# A better product grid for product models and variants
```
Comment (LaureBro) [inline content/updates/2020-10/3-A-better-product-grid-for-product-models-and-variants.md:9]:
```suggestion
# Product grid improvements for product models and variants
```

File: content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Jobs and process tracker multiples improvements
```
Comment (LaureBro) [inline content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md:9]:
```suggestion
# Jobs and process tracker improvements
```
Are those improvements related to the 1st article "# Interrupt your imports and exports" (that I want to rename into "# More control over imports and exports" maybe we should put everything under this article?

File: content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+## Progress bar and time left for your in-progress jobs
```
Comment (LaureBro) [inline content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md:11]:
```suggestion
## Progress bar and time left for your ongoing jobs
```

File: content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/.html"
+:::
+
+## Easy access to the detail of your quick export once launched
```
Comment (LaureBro) [inline content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md:15]: I didn't understand the feature, which detail is accessible?

File: content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+::: meta-data type="Improvement" features="Productivity" available="mid-October" link-to-doc="../articles/.html"
+:::
+
+## Process tracker grid quick-win enhancements
```
Comment (LaureBro) [inline content/updates/2020-10/6-Job-and-process-tracker-multiple-improvements.md:19]:
```suggestion
## Process tracker display enhancements
```

File: content/updates/2020-10/5-Up-to-300-measurements-families.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Up to 300 measurements families
```
Comment (LaureBro) [inline content/updates/2020-10/5-Up-to-300-measurements-families.md:9]:
```suggestion
# Number of measurement families limit extended to 300
```

File: content/updates/2020-10/4-Data-Quality-Insights-new-criteria-and-ability-to-choose-your-data-quality-focuses.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+# Data Quality Insights new criteria and ability to choose your quality focuses
```
Comment (LaureBro) [inline content/updates/2020-10/4-Data-Quality-Insights-new-criteria-and-ability-to-choose-your-data-quality-focuses.md:7]: Title will be probably too long, it's 77 char included spaces.
The word 'criteria' makes me think of enrichment and consistency... I'd probably need more details on the features to explain them :)

File: content/updates/2020-10/1-Interrupt-your-imports-and-exports.md (hunk 1 above)
Comment (NolwennP) [reply content/updates/2020-10/1-Interrupt-your-imports-and-exports.md:9]:
```suggestion
# Stop your imports and exports
```

File: content/updates/2020-10/3-A-better-product-grid-for-product-models-and-variants.md (hunk 4 above)
Comment (NolwennP) [reply content/updates/2020-10/3-A-better-product-grid-for-product-models-and-variants.md:9]:
```suggestion
# Better manage product models and variants in the grid
```

File: content/updates/2020-10/7-Up-to-300-measurements-families.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Up to 300 measurements families
```
Comment (NolwennP) [inline content/updates/2020-10/7-Up-to-300-measurements-families.md:9]:
```suggestion
# Measurements families limit increased to 300
```

File: content/updates/2020-10/5-DQI-new-missing-image-criteria.md [hunk 12]
```diff
@@ -0,0 +1,10 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+
+# Data Quality Insights : New "Missing image" criteria
```
Comment (NolwennP) [inline content/updates/2020-10/5-DQI-new-missing-image-criteria.md:8]:
```suggestion
# Data Quality Insights: New "Missing image" criteria
```

File: content/updates/2020-10/5-DQI-new-missing-image-criteria.md [hunk 13]
```diff
@@ -0,0 +1,10 @@
+---
+pim_announcement_img: ./img/.png
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+
+# Data Quality Insights: New "Missing image" criteria
```
Comment (LaureBro) [inline content/updates/2020-10/5-DQI-new-missing-image-criteria.md:8]:
```suggestion
# Data Quality Insights new "Missing image" criteria
```

**Review by LaureBro — APPROVED** [review]

File: content/updates/2020-10/3-Better-manage-product-models-and-variants-in-the-grid.md [hunk 14]
```diff
@@ -0,0 +1,26 @@
+---
+pim_announcement_img: ./img/.png
```
Comment (NolwennP) [inline content/updates/2020-10/3-Better-manage-product-models-and-variants-in-the-grid.md:2]:
```suggestion
```

File: content/updates/2020-10/2-Automate-your-description-creation-with-rules.md [hunk 15]
```diff
@@ -0,0 +1,11 @@
+---
+pim_announcement_img: ./img/.png
```
Comment (NolwennP) [inline content/updates/2020-10/2-Automate-your-description-creation-with-rules.md:2]:
```suggestion
```

File: content/updates/2020-10/1-Stop-your-imports-and-exports.md [hunk 16]
```diff
@@ -0,0 +1,11 @@
+---
+pim_announcement_img: ./img/.png
```
Comment (NolwennP) [inline content/updates/2020-10/1-Stop-your-imports-and-exports.md:2]:
```suggestion
```

File: content/updates/2020-10/4-Apply-Data-Quality-Insights-on-selected-attribute-groups.md [hunk 17]
```diff
@@ -0,0 +1,10 @@
+---
+pim_announcement_img: ./img/.png
```
Comment (NolwennP) [inline content/updates/2020-10/4-Apply-Data-Quality-Insights-on-selected-attribute-groups.md:2]:
```suggestion
```

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md [hunk 18]
```diff
@@ -0,0 +1,21 @@
+---
+pim_announcement_img: ./img/.png
```
Comment (NolwennP) [inline content/updates/2020-10/6-Job-and-process-tracker-improvements.md:2]:
```suggestion
```

File: content/updates/2020-10/5-DQI-new-missing-image-criteria.md [hunk 19]
```diff
@@ -0,0 +1,10 @@
+---
+pim_announcement_img: ./img/.png
```
Comment (NolwennP) [inline content/updates/2020-10/5-DQI-new-missing-image-criteria.md:2]:
```suggestion
```

File: content/updates/2020-10/7-Measurement-families-limit-increased-to-300.md [hunk 20]
```diff
@@ -0,0 +1,11 @@
+---
+pim_announcement_img: ./img/.png
```
Comment (NolwennP) [inline content/updates/2020-10/7-Measurement-families-limit-increased-to-300.md:2]:
```suggestion
```

File: content/updates/2020-10/8-New-filters-for-rules-conditions.md [hunk 21]
```diff
@@ -0,0 +1,10 @@
+---
+pim_announcement_img: ./img/.png
```
Comment (NolwennP) [inline content/updates/2020-10/8-New-filters-for-rules-conditions.md:2]:
```suggestion
```

File: content/updates/2020-10/1-Stop-your-imports-and-exports.md [hunk 22]
```diff
@@ -0,0 +1,10 @@
+---
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Stop your imports and exports
+::: meta-data type="New" features="Productivity" available="late October" link-to-doc="../articles/.html"
+:::
```
Comment (NolwennP) [inline content/updates/2020-10/1-Stop-your-imports-and-exports.md:10]:
```suggestion
:::

TODO
```

File: content/updates/2020-10/2-Automate-your-description-creation-with-rules.md [hunk 23]
```diff
@@ -0,0 +1,9 @@
+---
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+
+# Automate your description creation with rules
+::: meta-data type="New" features="Automation, Productivity" available="early October" link-to-doc="../articles/.html"
+:::
```
Comment (NolwennP) [inline content/updates/2020-10/2-Automate-your-description-creation-with-rules.md:9]:
```suggestion
:::

TODO
```

File: content/updates/2020-10/3-Better-manage-product-models-and-variants-in-the-grid.md [hunk 24]
```diff
@@ -0,0 +1,25 @@
+---
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Better manage product models and variants in the grid
+
```
Comment (NolwennP) [inline content/updates/2020-10/3-Better-manage-product-models-and-variants-in-the-grid.md:9]:
```suggestion

TODO
```

File: content/updates/2020-10/4-Apply-Data-Quality-Insights-on-selected-attribute-groups.md [hunk 25]
```diff
@@ -0,0 +1,9 @@
+---
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+
+# Apply Data Quality Insights on selected attribute groups
+::: meta-data type="New" features="Data quality" available="mid-October" link-to-doc="../articles/.html"
+:::
```
Comment (NolwennP) [inline content/updates/2020-10/4-Apply-Data-Quality-Insights-on-selected-attribute-groups.md:9]:
```suggestion
:::

TODO
```

File: content/updates/2020-10/5-DQI-new-missing-image-criteria.md [hunk 26]
```diff
@@ -0,0 +1,9 @@
+---
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+
+# Data Quality Insights new "Missing image" criteria
+::: meta-data type="New" features="Data quality" available="late October" link-to-doc="../articles/.html"
+:::
```
Comment (NolwennP) [inline content/updates/2020-10/5-DQI-new-missing-image-criteria.md:9]:
```suggestion
:::

TODO
```

File: content/updates/2020-10/6-Job-and-process-tracker-improvements.md [hunk 27]
```diff
@@ -0,0 +1,20 @@
+---
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Jobs and process tracker improvements
```
Comment (NolwennP) [inline content/updates/2020-10/6-Job-and-process-tracker-improvements.md:8]:
```suggestion

TODO
```

File: content/updates/2020-10/7-Measurement-families-limit-increased-to-300.md [hunk 28]
```diff
@@ -0,0 +1,10 @@
+---
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Measurement families limit increased to 300
+::: meta-data type="New" features="Governance" available="mid-October" link-to-doc="../articles/.html"
+:::
```
Comment (NolwennP) [inline content/updates/2020-10/7-Measurement-families-limit-increased-to-300.md:10]:
```suggestion
:::

TODO
```

File: content/updates/2020-10/8-New-filters-for-rules-conditions.md [hunk 29]
```diff
@@ -0,0 +1,9 @@
+---
+pim_announcement_alt_img: ""
+pim_announcement_audience:
+- EE
+---
+
+# New filters for rules conditions
+::: meta-data type="New" features="Automation, Productivity" available="early October" link-to-doc="../articles/.html"
+:::
```
Comment (NolwennP) [inline content/updates/2020-10/8-New-filters-for-rules-conditions.md:9]:
```suggestion
:::

TODO
```

**Pull request merged**